services/task-svc/migrations/000005_init.up.sql

```diff
@@ -7,3 +7,13 @@ CREATE TABLE IF NOT EXISTS subtasks (
 	    REFERENCES tasks(id)
 		ON DELETE CASCADE
 );
+
+CREATE TABLE IF NOT EXISTS tasks (
```

Comment:
The table was already created in a past migration. Please use `ALTER TABLE` instead. [Here is an example in a draft pr of mine.](https://github.com/helpwave/services/pull/154/files#diff-b6198b7b4f8d26fd48d3270adb316a84135c555e75a12f522cf87e1533c9e6d7R4)

Think about starting the migrations from 0 to 5. When the table gets created at migration 0, the new attributes will not be added due to the `CREATE TABLE IF NOT EXISTS` condition, because the table already exists.